addons/test_website/tests/test_reset_views.py

```diff
@@ -17,8 +18,14 @@ def fix_it(self, page, mode='soft'):
         resp = self.url_open(page)
         self.assertEqual(resp.status_code, 500, "Waiting 500")
         self.assertTrue('<button data-mode="soft" class="reset_templates_button' in resp.text)
-        data = {'view_id': self.find_template(resp), 'redirect': page, 'mode': mode}
-        resp = self.url_open('/website/reset_template', data)
+        data = {
+            'params':{
+                'view_id': self.find_template(resp),
+                'mode': mode
+                }
+            }
```

Comment:
Multiple linter errors :grimacing:
And missing `,`
```suggestion
        data = {
            'params': {
                'view_id': self.find_template(resp),
                'mode': mode,
            }
        }
```

Note that it would very well fit on a single line (it's weird to split this line but not the next one which is longer, but it's nitpicking)
```py
        data = {'params': {'view_id': self.find_template(resp), 'mode': mode}}
```
Which would probably even ease/reduce the diff as only part of the existing line will be shown as diff